deployment/azureDeploy.json

```
@@ -24,7 +24,7 @@
             "defaultValue": "https://azure.com",
             "type": "string",
             "metadata": {
-                "description": "Default URL use when the key pass by the user is not found."
+                "description": "Default URL use when key is passed by the user is not found."
```

Unfortunately that is still not correct grammar. I think this is the correction you were going for- just take out the is you added:
```suggestion
                "description": "Default URL use when key passed by the user is not found."
```